include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -3087,9 +3087,8 @@ namespace internal
       set_active_fe_index (const DoFCellAccessor<dealii::hp::DoFHandler<dim,spacedim>, level_dof_access> &accessor,
                            const unsigned int                                      i)
       {
-        typedef dealii::DoFAccessor<dim,DoFHandler<dim,spacedim>, level_dof_access> BaseClass;
         Assert (accessor.dof_handler != nullptr,
-                typename BaseClass::ExcInvalidObject());
+                (typename dealii::DoFAccessor<dim,DoFHandler<dim,spacedim>, level_dof_access>::ExcInvalidObject()));
```

Comment:
Should this be an `hp::DoFHandler`?

Reply:
Apparently not: The `hp` version of this class does not have `ExcInvalidObject()`. I see that original typedef in this function was `typedef dealii::DoFAccessor<dim,DoFHandler<dim,spacedim>, level_dof_access> BaseClass`.

Reply:
I think he meant `dealii::DoFAccessor<dim,hp::DoFHandler<dim,spacedim>, level_dof_access>`. But it ends up with the same exception class, of course.